_layouts/home.html

```diff
@@ -3,24 +3,28 @@
 ---
 
 <div class="home">
+  {% if page.title %}
```

Comment:
Its a slot to insert an optional `<h1>` to the landing page where heading `!= {{ site.title }}`.

e.g.

--
# Blah Chronicles | (site title) |

## The Spiral Consortium | (h1 page title) |

Lorem ipsum dolor sit amet and all that random stuff.
But before we move on, here's a list of all the posts in this site:

### Posts | (h2 list title) |

  - **Jun 27, 2017**
    **Welcome to the Consortium**
    Welcome cadet! By the end of this post, you'll be familiar about what we do here...
  - **Jun 28, 2017**
    **The Alpha**
    She may not look that menacing, but cross her path wrongly and you shall feel her wrath...